test/support/test_config.h

```diff
@@ -54,6 +54,12 @@
 #define _PSTL_STD_UNINITIALIZED_FILL_BROKEN (_MSC_VER == 1900)
 // GCC10 produces wrong answer calling exclusive_scan using vectorized polices
 #define TEST_GCC10_EXCLUSIVE_SCAN_BROKEN (_GLIBCXX_RELEASE == 10)
+// Array swap broken on Windows because Microsoft implementation of std::swap function for std::array
+// call some function which is not declared as SYCL external and we have compile error
+#define TEST_XPU_ARRAY_SWAP_BROKEN (_MSC_VER <= 1937)

```

Comment:
@kboyarinov on Linux we have compile error now at all places where we using this macro in tests: use of undeclared identifier '_MSC_VER'